### ISSUE TITLE
Concise release of assets 

### ISSUE THREAD

**Issue by fabioluciano** [microsoft/cascadia-code#322]
#Description of the new feature/enhancement (with images if possible)

The assets follow a pattern in the process of releasing a new version. The assets released until now, follow this patterns:
* A zip file with with ttf fonts + zip from tag;
* The fonts itself + zip from tag;
* A zip file with woff2, ttf and otf;

# Proposed technical implementation details (optional)
* Follow [semantic versioning](https://semver.org/) for tags
* Release the fonts in the assets outside of a tar/zip file, and we can use the zip option of github to download the "all-in-package'

**Comment by fabioluciano** [comment]
I'm tryng to automate the installing process of this, and its hard to follow all this different assets release process  :)

**Comment by DHowett** [comment]
Thanks for the request!

So, there have only been three iterations of the asset format, and we've never reverted from a newer format back to an older one. I'm not intending on changing the asset format again, because it seems like this archive is right at the intersection of good and useful.

1. One bare font file.
2. Multiple bare font files.
3. An archive of multiple formats of font file.

I like the idea of letting GitHub handle it, but GitHub's `zip` downloader lets you download an archive of the _source code_, not an archive of an entire release's assets. A release archive still eludes us, unless we prepare it and upload it ourselves.

As for versioning: there is no notion of compatibility in a font, and I do not believe that SemVer really applies to fonts. Take the first clause in the specification...

> _The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in RFC 2119._
>
> Software using Semantic Versioning *MUST* declare a public API.

emphasis mine. :smile:

**Comment by fabioluciano** [comment]
Hi @DHowett!

I'm Ok with this format :)

you are right about semantic versioning. There is any plan to release the fonts code?

**Comment by DHowett** [comment]
The whole font is open-source! All the code is in this repository.

**Comment by fabioluciano** [comment]
Heheheh

The last time i pull the repo is on the version v1909.16. Now i see the code.

Thank you again :kiss:
